qbft/spectest/tests/messages/invalid_prepare_justification_unmarshaling.go

```diff
@@ -1,8 +1,33 @@
 package messages
 
-import "github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
 
 // InvalidPrepareJustificationsUnmarshalling tests unmarshalling invalid prepare justifications (during message.validate())
 func InvalidPrepareJustificationsUnmarshalling() tests.SpecTest {
```

Comment:
@alonmuroch
It seems this is mainly testing ssz logic that is the repsonsibility of a 3rd party lib.

We can have this basic test but just wondering what is your view of testing such parts of the code?